examples/tensorrt_llm/components/worker.py

```diff
@@ -62,6 +65,24 @@ def __init__(self):
     async def async_init(self):
         self._init_engine()
 
+        runtime = dynamo_context["runtime"]
+        logger.info("Registering LLM for discovery")
+        comp_ns, comp_name = TensorRTLLMWorker.dynamo_address()  # type: ignore
+        endpoint = runtime.namespace(comp_ns).component(comp_name).endpoint("generate")
+
+        try:
+            await register_llm(
+                ModelType.Backend,
+                endpoint,
+                self._engine_config.model_name,
+                self.served_model_name,
+                kv_cache_block_size=self._kv_block_size,
```

Comment:
both chat completion and completion will be registered?

Reply:
ModelType.Backend here: https://github.com/ai-dynamo/dynamo/blob/8f42f23cd6faf1ff735ba393cc96e9adcdd9c3b0/examples/tensorrt_llm/components/worker.py#L75
means that we want to use pre/post processor from rust and hence it would expose both the endpoints. 
This engine only see tokens and should not care about the chat/completion distinction.